packages/spright-components/.prettierignore

```diff
@@ -0,0 +1,29 @@
+# Visit and ignore all objects in the root. 
+# Then explicitly allow all TypeScript, JavaScript, Markdown, and JSON files in the root, src, and .storybook folders.
+/*
+!*.ts
+!*.js
+!*.md
+!*.mdx
+!*.json
+!src/
+!/.storybook/
```

Comment:
Even though this PR doesn't include a Storybook, I figured it wouldn't be worth the bother to remove all the Storybook-related configuration just to add it back later.